sound/hybrid_font.h

```diff
@@ -193,16 +208,41 @@ class HybridFont : public SaberBase {
   void SB_EndLockup() override {
     if (lock_player_) {
       // Polyphonic case
+
+      // ultra - dynalock lockoff
+
+      if (lockoff.files_found()) {
+        PlayPolyphonic(&lockoff);
+      } else {
+        if (config_.clashLockOff) { // ultra clash fallback
+          PlayPolyphonic(&clsh);
+        }
+      }
+
+      // ultra
+	    
```

Comment:
If lockoff is present, we need to change the fade time below to something much shorter.